src/Streamly/System/Process.hs

```diff
@@ -85,13 +84,13 @@ openProc ::
     -> IO (Handle, ProcessHandle)
     -- ^ Handle to read from output of process, process handle
 openProc fpath args = do
-    (readEnd, writeEnd) <- createPipe
     let procObj = (proc fpath args) {
-            std_out = UseHandle writeEnd,
-            close_fds = True
+            std_out = CreatePipe,
```

Comment:
The other handles i.e. `std_in` and `std_err` are set to inherit from the parent process, is this desirable?

Reply:
That sounds correct.